x-pack/plugin/ccr/qa/security/src/test/java/org/elasticsearch/xpack/ccr/FollowIndexSecurityIT.java

```diff
@@ -66,26 +68,26 @@ public void testFollowIndex() throws Exception {
         } else {
             followIndex(client(), "leader_cluster", allowedIndex, allowedIndex);
             assertBusy(() -> verifyDocuments(allowedIndex, numDocs, "*:*"));
-            assertThat(countCcrNodeTasks(), equalTo(1));
+            assertThat(getCcrNodeTasks(), contains(new CcrNodeTask("leader_cluster", allowedIndex, allowedIndex, 0)));
```

Comment:
This assertion is currently failing with `Expected: <1> but: was <2>`.
I would like to know what is the extra element to be able to troubleshot this further, however this test is not reproducing locally. I am hoping this extra info will help to understand the source of the issue once the failure happens again.